roles/ceph-mon/tasks/ceph_keys.yml

```diff
@@ -99,13 +99,32 @@
     - "{{ ceph_keys.get('stdout_lines') | default([]) }}"
   when: cephx
 
+- name: check if libvirt keyring exists
+  command: find /etc/ceph -name "{{ cluster }}.client.libvirt.keyring"
+  changed_when: false
+  register: is_libvirt_keyring_exists
+  always_run: true
+  run_once: true
+  when: cephx
+
+- name: set ceph libvirt keyring permission
+  shell: chown ceph:ceph *.libvirt.keyring ; chmod 0640 *.libvirt.keyring
```

Comment:
Previously you were using the `file` module which is exactly what you should be doing.
My last comment was the owner/mode might need to be configurable with variables so this is not hardcoded.